package/MDAnalysis/analysis/__init__.py

```diff
@@ -27,6 +27,20 @@
 
 The :mod:`MDAnalysis.analysis` sub-package contains various recipes and
 algorithms that can be used to analyze MD trajectories.
+If not stated differently, an analysis conducted by the available modules
+always follows the same structure
+
+1. Initialize the object previously imported.
+2. Run the analysis for specific trajectory slices
+3. Access the analysis from the `results` attribute (if available)
```

Comment:
I thought the point of this PR was to standardize things so it was always available?

Reply:
No it wasn't I added these lines.